internal/capability/audit.go

```diff
@@ -35,6 +37,10 @@ type capAudit struct {
 	// all of them must be an implemented method of capAudit and must be part
 	// of the Audit interface
 	auditPlan []string
+
+	// customResources is a map of string interface that has all the CR(almExamples) that needs to be installed
```

Comment:
I think it would be nice to explain here that this can be ALM Examples or any other CRs. I think we should be able to inject CRs as part of a parameter to test the operator. That's certainly not part of this PR but a future one. Check #130